cmless.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name        = 'cmless'
-  s.version     = '0.0.8'
-  s.date        = '2015-09-16'
+  s.version     = '0.0.9'
```

Comment:
@mccalluc is there a reason this isn't on 0.1.0 yet? Semver recommends using right-most digit for bug fixes, the middle digit for non-breaking enhancements, e.g. TOC functionality.